### DISCUSSION TITLE
Issue Setting Up

### DISCUSSION THREAD

**Discussion by jdbhartley** [wolveix/satisfactory-server#10] [Q&A] [answered]
So I created a directory '/home/user/config' and git cloned your repo to '/home/user' then ran _**docker run -it steamcmd/steamcmd +login Your-Steam-Username Your-Steam-Password**_ and logged in successfully (I'm using steamguard authenticator) 

After than I ran the 

_**docker run -d --name=satisfactory-server -h satisfactory-server -e STEAMUSER=Your-Steam-Username -e STEAMPWD=Your-Steam-Password -e STEAMCODE=Your-Steam-Code -v /path/to/config:/config -p 7777:7777/udp wolveix/satisfactory-server:latest**_

It downloads and extracts some files, then.. nothing. I feel like im missing some steps here

**Comment by wolveix** [comment, marked as answer]
Hey! Thanks for trying this repo :) 

Could you post your full log from the Docker container, please? 

`docker logs satisfactory-server`

**Reply by jdbhartley** [reply]
Thank you for the quick response. here is the log

> Downloading the latest version of the game...
Redirecting stderr to '/root/.steam/logs/stderr.txt'
[  0%] Checking for available updates...
[----] Download complete.
[----] Installing update...
[----] Extracting package...
[----] Extracting package...
[----] Extracting package...
[----] Extracting package...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Cleaning up...
[----] Update complete, launching Steamcmd...
Redirecting stderr to '/root/.steam/logs/stderr.txt'
[  0%] Checking for available updates...
[----] Verifying installation...
Steam Console Client (c) Valve Corporation
-- type 'quit' to exit --
Loading Steam API...OK.
[1m@sSteamCmdForcePlatformType windows
[0m"@sSteamCmdForcePlatformType" = "windows"
[0mLogging in user 'hayloe' to Steam Public ...
Logged in OK
Waiting for user info...OK
[1mforce_install_dir /config/gamefiles
[0m[1mapp_update "526870"
[0m Update state (0x3) reconfiguring, progress: 0.00 (0 / 0)
Error! App '526870' state is 0x202 after update job.
[1mquit
[0m * Starting periodic command scheduler cron
   ...done.
Game binary is missing.

>

**Reply by wolveix** [reply]
Thanks for providing the log! Per the end of the log, the game download/update failed. According to this [link](https://oxidemod.org/threads/error-app-state-is-0x202-after-update-job-out-of-space.17980/), the error is due to you not having enough space free to download the game (you need approximately 16GB free).

**Reply by jdbhartley** [reply]
:D I figured that out, was setting up my VM with too little space. Thank you for the help! It's all up and running now.

Strange problem though, I'm unable to pickup any leaves or wood laying around, ever experience this?

**Reply by jdbhartley** [reply]
looks like a common multiplayer bug.

**Reply by wolveix** [reply]
Yeah, unfortunately there are a few common multiplayer bugs given that the game is in early access. I've just added a storage check into the container on runtime to help guide others in the future :) Thank you for your feedback!
